providers/hubspot/webhook.go

```diff
@@ -0,0 +1,47 @@
+package hubspot
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+type WebhookMessage struct {
+	AppId            int    `json:"appId"`
+	EventId          int    `json:"eventId"`
+	SubscriptionId   int    `json:"subscriptionId"`
+	PortalId         int    `json:"portalId"`
+	OccurredAt       int    `json:"occurredAt"`
+	SubscriptionType string `json:"subscriptionType"`
+	AttemptNumber    int    `json:"attemptNumber"`
+	ObjectId         int    `json:"objectId"`
+	ChangeSource     string `json:"changeSource"`
+	PropertyName     string `json:"propertyName"`
+	PropertyValue    string `json:"propertyValue"`
+}
+
+type WebhookResult struct {
```

Comment:
@laurenzlong I wasn't too sure if this return type makes sense. I think we do need to get the original record. The way we standardize in `Read` is to use read result. but `GetRecord` is just reading a single record and not sure using the same structure makes sense. 

Maybe it might be better to introduce another type 
```
type Record struct {
   RecordId string
   Data any
}
```

or an interface

type Record interface {
  RecordId() string
  Raw any
}